Windows tests are failing

Several of the recent PRs have Windows tests failing with a stack overflow:
```
Error: Error: C stack usage  2101032 is too close to the limit
error: test failed, to rerun pass '--lib'
Error: i686-pc-windows-gnu
```
For example: https://github.com/extendr/extendr/pull/128/checks?check_run_id=1705514593#step:8:95

Since this seems to be happening with all recent PRs, I suspect it's not related to them but some other cause.

Oh, but can I pass `-1`...?

```
   Compiling libR-sys v0.2.0
   Compiling extendr-engine v0.1.11 (/home/yutani/repo/extendr/extendr-engine)
error[E0600]: cannot apply unary operator `-` to type `usize`
  --> extendr-engine/src/lib.rs:47:29
   |
47 |             R_CStackLimit = -1;
   |                             ^^
   |                             |
   |                             cannot apply unary operator `-`
   |                             help: you may have meant the maximum value of `usize`: `usize::MAX`
   |
   = note: unsigned values cannot be negated

error: aborting due to previous error
```

--- a/extendr-engine/src/lib.rs
+++ b/extendr-engine/src/lib.rs
@@ -43,10 +43,7 @@ pub fn start_r() {
             // In case you are curious.
             // Maybe 8MB is a bit small.
             // eprintln!("R_CStackLimit={:016x}", R_CStackLimit);
-
-            if cfg!(not(target_os = "windows")) {
-                R_CStackLimit = usize::max_value();
-            }
+            R_CStackLimit = usize::max_value();
 
             setup_Rmainloop();
         }

